Python/suggestions.c

```diff
@@ -43,7 +43,7 @@ levenshtein_distance(const char *a, size_t a_size,
                      const char *b, size_t b_size,
                      size_t max_cost)
 {
-    static size_t buffer[MAX_STRING_SIZE];
+    size_t buffer[MAX_STRING_SIZE] = {0};
```

Comment:
We can move the declare this code to the line 74 as `size_t buffer[a_size] = {0};`

Reply:
`msvc` does not supports vla so we cannot use it.